roles/common/files/letsencrypt-gencert

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+service apache2 stop
```

Comment:
@mikeashley Yeah, on second thought you're right. I was thinking that Apache was somehow intrinsically related to this set of tasks, but really it isn't; there's no particular reason for this task file to be responsible for ensuring that it's running. I'll remove it.